platform/commons/src/main/java/com/peregrine/commons/ResourceUtils.java

```diff
@@ -30,6 +30,7 @@
             JcrConstants.JCR_PREDECESSORS,
             JcrConstants.JCR_UUID,
             JcrConstants.JCR_VERSIONHISTORY,
+            JcrConstants.JCR_PREDECESSORS,
```

Comment:
Isn't this redundant? The same constant is defined above already

Reply:
Yes, it is, thank you for noticing :)